pkg/webhook/webhookreporter.go

```diff
@@ -71,14 +71,11 @@ func (r *WebhookReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 func (r *WebhookReconciler) reconcileReport(reportType client.Object) reconcile.Func {
 	return func(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+		err := r.Client.Get(ctx, request.NamespacedName, reportType)
```

Comment:
error check should be done right after `client get call` , I would return `L74`
`err := r.Client.Get(ctx, request.NamespacedName, reportType)` back to L81

Reply:
Sure, moving polishing things to keep the code more readable.